x-pack/plugins/security_solution/public/detections/components/rules/description_step/index.tsx

```diff
@@ -220,6 +238,11 @@ export const getDescriptionItem = (
       return [];
     }
   } else if (field === 'suppressionMissingFields') {
+    // only query rule can have suppression missing fields
+    const ruleType: Type = get('ruleType', data);
+    if (!isQueryRule(ruleType)) {
```

Comment:
Similar to the above:

```suggestion
    const ruleType: Type = get('ruleType', data);
    const ruleCanHaveSuppressionMissingFields =  isQueryRule(ruleType);
    if (!ruleCanHaveSuppressionMissingFields) {
```